projects/analog-phytocell/index.md

```diff
@@ -0,0 +1,149 @@
+---
+authors:
+  - "Natalie Schreiber"
+layout: project
+title: "Projekt 1 | Algorithmus | Phytocell"
+type: project
+---
+
+![Cover](splash.png)
+
+# Analog Algorithm | PhytoCell
+
+## General
+
+_Algorithms? That reminds me of confusing and therefore boring math lessons in school.
+I'm done with it._
+
+You're not.
```

Comment:
No slang and twitter, facebook writing style please… It is not a blog. It is a documentation of your work. Try a objective perspective on the project.